language/move-bytecode-verifier/src/code_unit_verifier.rs

```diff
@@ -28,26 +28,33 @@ pub struct CodeUnitVerifier<'a> {
 }
 
 impl<'a> CodeUnitVerifier<'a> {
-    pub fn verify_module(module: &'a CompiledModule) -> VMResult<()> {
-        Self::verify_module_impl(module).map_err(|e| e.finish(Location::Module(module.self_id())))
+    pub fn verify_module(module: &'a CompiledModule, max_loop_depth: usize) -> VMResult<()> {
```

Comment:
Would it make more sense to just pass in the config?

Reply:
Sounds like a good idea. This way we won't have to touch the argument passing part again if we want to add another one in the future.